jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -641,10 +600,8 @@ void delete_requestTenantIdMatchesResourceTenantId_noExceptionThrown() throws Js
     @Test
     @Transactional
     void delete_requestTenantIdDoesNotMatchResourceTenantId_resourceNotFoundExceptionThrown() throws JsonProcessingException {
-        String payload = "{" +
-                "            \"" + DESCRIPTION_FIELD_NAME + "\": \"Dummy Entity C 100\"," +
-                "            \"" + INDEX_FIELD_NAME + "\": 1" +
-                "        }";
+        String payload = PayloadCreator.createIDPayload(List.of(DESCRIPTION_FIELD_NAME, INDEX_FIELD_NAME),
```

Comment:
As per comment in PayloadCreator, this could then become
```
String payload = Map.of(
                DESCRIPTION_FIELD_NAME, "Dummy Entity C 100",
                INDEX_FIELD_NAME, "1"
        );
```

Reply:
I've moved this to using a map now